### ISSUE TITLE
Implement JRE generation task

### ISSUE THREAD

**Issue by chippmann** [utopia-rise/godot-kotlin-jvm#163]
Best would be a task triggered by the Editor plugin which generates the jre when a new kotlin script is created (similar to the C# plugin) or when run is pressed but no jre is present (for the case when scripts are solely created outside the editor with Intellij for example). Also some good error messages would be nice (for example no JAVA_HOME found, JAVA <= 9 installed, and so on...).

Another possibility could be to handle this with a gradle task which get's called by the editor plugin.

Best might be to have both and execute the gradle task upon build and it checks on each build if a jre is present. So we cover the case where the godot editor is never started.

**Comment by piiertho** [comment]
I agree we should make it as a gradle task, this would be easier for packaging games from CI
